src/main/java/baritone/process/MineProcess.java

```diff
@@ -188,10 +188,10 @@ private PathingCommand updateGoal() {
             return new PathingCommand(goal, legit ? PathingCommandType.FORCE_REVALIDATE_GOAL_AND_PATH : PathingCommandType.REVALIDATE_GOAL_AND_PATH);
         }
         // we don't know any ore locations at the moment
-        if (!legit) {
+        if (!Baritone.settings().exploreForBlocks.value) {
```

Comment:
Perhaps this could be `if (!legit && !Baritone.settings().exploreForBlocks.value) {`

Would that still work for what you're trying to do with this PR?

Reply:
Yes, the only case I definitely want to change is `exploreForBlocks true` + `legitMine false`, which would still work.

Reply:
Okay let's do that then. I can see the benefit of that and that makes sense, and this way it won't have too bad of backwards incompatibility. 👍